common/common.go

```diff
@@ -30,6 +30,16 @@ var (
 	version      = "dev"
 )
 
+const (
+	patternALB = `(?P<type>[^ ]+) (?P<timestamp>[^ ]+) (?P<elb>[^ ]+) (?P<client>[^ ]+) (?P<target>[^ ]+) (?P<request_processing_time>[^ ]+) (?P<target_processing_time>[^ ]+) (?P<response_processing_time>[^ ]+) (?P<elb_status_code>[^ ]+) (?P<target_status_code>[^ ]+) (?P<received_bytes>[^ ]+) (?P<sent_bytes>[^ ]+) "(?P<request>[^"]+)" "(?P<user_agent>[^"]+)" (?P<ssl_cipher>[^ ]+) (?P<ssl_protocol>[^ ]+) (?P<target_group_arn>[^ ]+) "Root=(?P<trace_id>[^"]+)" "(?P<domain_name>[^"]+)" "(?P<chosen_cert_arn>[^"]+)" (?P<matched_rule_priority>[^ ]+) (?P<request_creation_time>[^ ]+) "(?P<actions_executed>[^"]+)" "(?P<redirect_url>[^"]+)" "(?P<error_reason>[^"]+)" "(?P<target_list>[^"]+)" "(?P<target_status_code_list>[^"]+)"`
```

Comment:
I know you just lifted these straight from YML files, but now that they're embedded in the Go code it feels like it would be useful to document them and possibly add some tests to validate them.

In terms of readability and maintainability, did you consider using a pattern builder helper function?. A couple of these could literally be a list of names and a single format string.

At the very least, is it worth considering breaking this up into a set of concatenated substrings with 
line breaks?